src/pages/PrepareCondition/index.tsx

```diff
@@ -809,8 +809,8 @@ export const PrepareCondition = () => {
           </ErrorContainer>
         )}
         {newCustomConditionStatusInfo &&
-          !customConditionFormHasErrors &&
-          !omenConditionFormHasHerros && (
+          ((conditionType === ConditionType.custom && !customConditionFormHasErrors) ||
+            (conditionType === ConditionType.omen && !omenConditionFormHasHerros)) && (
```

Comment:
omenConditionFormHasHerros is this a typo ?

Reply:
Sure.